WordPressAuthenticator/Analytics/AuthenticatorAnalyticsTracker.swift

```diff
@@ -255,7 +255,7 @@ public class AuthenticatorAnalyticsTracker {
         
         return Configuration(
             appleEnabled: WordPressAuthenticator.shared.configuration.enableUnifiedApple,
-            googleEnabled: WordPressAuthenticator.shared.configuration.enableUnifiedGoogle,
+            googleEnabled: true,
```

Comment:
@diegoreymendez I just realized these should be `WordPressAuthenticator.shared.configuration.enableUnifiedAuth`. I'll correct them in my next PR.

Reply:
Well... it's not so bad as long as unified auth is on, but I think you're right.